Fixes segmentation fault on OpenBSD 7.8 and greater.
The type for 'malloc_options' has changed to an immutable char pointer.
Better to not set it at all.

Index: src/main.cpp
--- src/main.cpp.orig
+++ src/main.cpp
@@ -659,7 +659,7 @@ main(int argc, char *argv[])
 {
 	char *cp;
 
-#if __OpenBSD__
+#if defined(OpenBSD) && OpenBSD <= 202504
 	extern char *malloc_options;
 
 	malloc_options = const_cast<char *>("S");
